features/log-analyzer/org.wso2.carbon.analytics.shared.la.common.feature/src/main/resources/gadgets/Log_Error_Bar_Chart/js/main.js

```diff
@@ -109,10 +109,10 @@ function initialize() {
             }
         }
     }, function (error) {
-        if(error === undefined){
+        if (error === undefined) {
             onErrorCustom("Analytics server not found.", "Please troubleshoot connection problems.");
             console.log("Analytics server not found : Please troubleshoot connection problems.");
-        }else{
+        } else {
             error.message = "Internal server error while data indexing.";
             onError(error);
             console.log(error);
```

Comment:
This console logs removed in following commit : https://github.com/wso2/shared-analytics/commit/f1f9a8cf9493cfc0ad276b355d4a1d6144ae7a83